### DISCUSSION TITLE
运行隐语SecretFlow实际场景MPC算法开发实践的时候出错

### DISCUSSION THREAD

**Discussion by YnRen22852** [secretflow/secretflow#1421] [General]
![image](https://github.com/user-attachments/assets/c262b6f5-9671-4816-95f0-d8420b2e41de)
在使用数据集https://file.io/qJwaEFiMdhzU运行https://secretflow.readthedocs.io/zh-cn/stable/tutorial/practical_case_walkthrough_using_sf_with_spu.html时出现上图错误

**Comment by YnRen22852** [comment]
数据集是 https://file.io/qJwaEFiMdhzU

**Comment by YnRen22852** [comment]
运行的场景是 https://secretflow.readthedocs.io/zh-cn/stable/tutorial/practical_case_walkthrough_using_sf_with_spu.html

**Reply by 383004576** [reply]
您目前运行的场景文档版本比较老，建议参考最新版本的，地址：https://www.secretflow.org.cn/zh-CN/

**Comment by 383004576** [comment]
您好，这个问题多数情况是节点通讯异常导致的，我们可能需要您提供详细的版本环境以及步骤来深入了解，问题沟通，请移步issue区：https://github.com/secretflow/secretflow/issues
